nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2743,6 +2743,64 @@ nifi.content.repository.encryption.key.id=Key1
 nifi.content.repository.encryption.key=0123456789ABCDEFFEDCBA98765432100123456789ABCDEFFEDCBA9876543210
 ....
 
+
+=== HDFS Content Repository Properties
```

Comment:
it should be highlighted this is a new and experimental implementation.  Also it should not be included in the default convenience binaries at this point so these docs should be elsewhere such as a readme or something like that.

Reply:
Sounds, good. Are there examples of this anywhere you can point me to?

Reply:
we dont have examples of external things I can point you to.  We have included things and been clear they're experimental.  This means they might not work.  We dont know how the behave.  And we might not keep them in subsequent release even. If this new nar is really small we can probably include it.  I've not built this myself to check

Reply:
Moving this documentation to a README.txt in the bundle home directory.